CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Pin `architect` binary version to v2.1.3.
```

Comment:
```suggestion
### Added

- Update `architect` binary version to v2.1.3.
```

Reply:
Why added? I don't get it. There is update of orb below in the changelog and it is under the changed section

Reply:
There is nothing really changed, everything is backward compatible. Now the orb supports 4 digit releases (added feature). We don't even have to mention architect got updated here. It isn't relevant to the user.